docs/generate.js

```diff
@@ -0,0 +1,16 @@
+const jsdoc2md = require('jsdoc-to-markdown');
+const fs = require('fs');
+
+const template = fs.readFileSync('./docs/template.hbs', 'utf-8');
+const opFactory = file => ({template, files: `./lib/${file}`});
+
+const files = fs.readdirSync('./lib', {withFileTypes: true});
```

Comment:
This is also picking up env.js which doesn't need docs.  Maybe easier to use an explict list of filenames to process?

Reply:
It picks it up, but if there are no js-doc comments in a file then no markdown file is generated. So I think it is actually kind of an advantage. if env.js just js-doc annotations they would be generated to markdown automatically.

Reply:
`docs/env.md` is in this commit.

Reply:
good catch looks like the template function produces an empty document where before it was an empty string so will look into this in a just a second.

Reply:
@dlongley ok I figured out a way. if the resulting doc's length is smaller than the template then no content was inserted. Failed doc files will always be smaller than the template because the template variables are replaced with white space.

Reply:
@davidlehn ok so the above check I mentioned is there, however node fs does not support globs. So I wrote a smaller regex so we only generate docs if a file ends with keypair.js or index.js. Is that enough? mark as resolved if so. The solutions for this can be pretty various so relying on you to indicate you are good with this solution. any other changes that have not been marked as resolved means my solution was not exactly what was suggested.